lib/cn/cn.c

```diff
@@ -449,7 +449,6 @@ cn_mblocks_destroy(
  *    @wlen  = 10.0 GiB
  *    @flags = CN_MB_EST_FLAGS_PREALLOC;
  */
-/* MTF_MOCK */
```

Comment:
Were these removed for a reason?

Reply:
`/* MTF_MOCK */` annotations mean nothing in `.c` files.

Reply:
Good catch! Didn't see the extension